.teamcity/src/VcsRoots.kt

```diff
@@ -1,24 +1,42 @@
+package subprojects
```

Comment:
Maybe we should move the file `VcsRoots.kt` to the `subprojects` directory

Reply:
Thought it was. My bad.